platform/linux-generic/include/odp_packet_io_internal.h

```diff
@@ -174,50 +97,16 @@ struct pktio_entry {
 	} out_queue[PKTIO_MAX_QUEUES];
 };
 
-typedef union {
+union pktio_entry_un {
```

Comment:
pktio_entry_u would be more standard than _un here. Not clear why you want to remove the typedef here.